vllm_flagos/ops/layernorm.py

```diff
@@ -0,0 +1,24 @@
+from typing import Optional, Union
+import torch
+from vllm.model_executor.layers.layernorm import RMSNorm
+from flag_gems.modules.normalization import gems_rms_forward
+
+class FlagOSRMSNorm(RMSNorm):
```

Comment:
It is advisable to rename FlagOSRMSNorm to RMSNormFL, and some similar names should also be revised.